### PR TITLE
Add useful switches in the readme

### PR DESCRIPTION
**Pull request by ambs**
Given the lack of knowledge on how to build the docker image, and as users as lazy to read documentation, I added a couple of items in the BUILD.md  to clarity.

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -15,6 +15,11 @@ $ git checkout master
 $ mvn -DskipTests package
 ```
 
+Useful build switches:
+- `-Ddocker=true` : builds the docker image
+- `-DskipTests` : skips running tests
+- `-Ddependency-check.skip=true` : skips validating dependencies
+
```
Comment (adamretter) [inline BUILD.md:22]: Could you just add under this:

```
Further build switches can be found at: http://www.exist-db.org/exist/apps/doc/exist-building#build-system
```

**Review by adamretter — COMMENTED** [review]
Just one small suggestion.

**Comment by ambs** [comment]
Did some rearrangements. Let me know if it looks good.

**Review by adamretter — APPROVED** [review]

**Comment by adamretter** [comment]
Thanks @ambs. Looks good :-)

**Pull request merged**